news/_posts/2020-call-for-paper-quick-tour.md

```diff
@@ -0,0 +1,57 @@
+
+Constantin Boulanger, MDWEB: "Intégration de PrestaShop au sein d'un ERP"
+<iframe width="560" height="315" src="https://www.youtube.com/embed/T_KX_Z7prZ0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Maxime Varinard, Vaisonet: "Webservice PrestaShop, bonnes pratiques et retour d'expérience"
+<iframe width="560" height="315" src="https://www.youtube.com/embed/0dsgu3dxzbA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Guillaume Batier, Prestasafe: "Améliorer les performances de sa boutique PrestaShop et optimiser le page speed"
+<iframe width="560" height="315" src="https://www.youtube.com/embed/T7oS5t9zFHg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Other interesting content
+
+Eric Senechal, PrestaShop's CTO, provided tech news and other information about the open source project:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QWt6DCb8Kr8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+And Pierre-Olivier Calande, PrestaShop's CPO, presented what's new in the upcoming 1.7.7 version of the software:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/WSP6xnx4QQc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+In parallel, the tech team at PrestaShop organised "Stream your tech", targeted to the developer community. It is a live streaming to share what they do and answer to questions. It has been recorded in two parts, listed below. And, as there has been great feedback about it, this kind of format will certainly be organised again (stay tuned). 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Az6v2qfsgpM" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/DqL1P6MWAC4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Conclusion
+
+The Call For Paper was quite new to organise, and difficult to maintain it in 2020, but overall, it was great to make it happen. And I really hope it will be possible to organise a new one next year. Please, share in the comments what you liked most this year, what could be improved, and what you would like to see in 2022.
```

Comment:
```suggestion
The Call For Papers was quite new to organise, and difficult to keep in 2020, but overall, it was great to make it happen. And I really hope it will be possible to organise a new one next year. Please, share in the comments what you liked the most this year, what could be improved, and what you would like to see in 2021.
```